### ISSUE TITLE
RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by hannesigrun** [StaticJsCMS/static-cms#674]
Jeg satt og tastet inn tekst (beskrivelse av resultater i resultatboks) og plutselig var det en feilmelding der....

**Applicable Versions:**
 - Static CMS version: ``
 - Git provider: `github`
 - Browser version: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36`

**CMS configuration**
```
local_backend: true
backend:
  name: github
  repo: mong/mongts
  branch: main
  base_url: https://prod-mong-api.skde.org
logo_url: https://www.skde.no/helseatlas/img/logos/helseatlas.svg
media_folder: /apps/skde/public/helseatlas/img
public_folder: /helseatlas/img
site_url: https://www.skde.no/helseatlas/
locale: nb_no
collections:
  - label: Atlas
    name: atlas
    sortable_fields:
      fields:
        - date
        - publisert
        - mainTitle
      default:
        field: date
        direction: Descending
    folder: apps/skde/_posts/helseatlas/atlas
    extension: json
    format: json
    create: true
    delete: false
    media_folder: /apps/skde/public/helseatlas/img/no/{{filename}}
    public_folder: /helseatlas/img/no/{{filename}}
    identifier_field: shortTitle
    editor:
      preview: false
    fields:
      - label: Filnavn
        name: filename
        widget: string
      - label: Publisert på forsiden
        name: publisert
        widget: boolean
        default: false
        hint: Atlaset legges på forsiden av skde.no/helseatlas hvis «Publisert på
          forsiden» er huket av.
      - label: Publiseringsdato
        name: date
        widget: datetime
      - label: Tittel
        name: mainTitle
        widget: string
      - label: Kort tittel
        name: shortTitle
        widget: string
      - label: Forsidebilde
        name: image
        widget: file
        required: true
      - label: Forsidetekst
        name: frontpagetext
        widget: string
        required: true
      - label: Ingress
        name: ingress
        widget: markdown
      - label: Språk
        name: lang
        widget: select
        options:
          - nb
          - nn
          - en
        default: nb
      - label: Kapittel
        name: kapittel
        widget: list
        fields:
          - label: Overskrift
            name: overskrift
            widget: string
            required: false
          - label: Innhold
            name: innhold
            widget: list
            hint: Selve innholdet i atlaset
            types:
              - label: Tekst
                name: tekst
                widget: object
                summary: "{{fields.beskrivelse}}"
                fields:
                  - label: Beskrivelse
                    name: beskrivelse
                    widget: string
                    hint: Kort beskrivelse, som kun vises som overskrift her (vil ikke vises på
                      nettsiden).
                    required: false
                  - label: Tekst
                    name: tekst
                    widget: markdown
                    hint: Seksjon med tekst og eventuelle statiske figurer
              - label: Faktaboks
                name: faktaboks
                widget: object
                summary: "{{fields.overskrift}}"
                fields:
                  - label: Overskrift
                    name: overskrift
                    widget: string
                    hint: Overskrift som vises på siden når resten av teksten er skjult
                  - label: Tekst
                    name: tekst
                    widget: markdown
                    hint: Faktabokstekst som i utgangspunktet er skjult men kan trykkes frem av
                      bruker.
              - label: Resultatboks
                name: resultatboks
                widget: object
                summary: "{{fields.overskrift}}"
                fields:
                  - label: Overskrift
                    name: overskrift
                    widget: string
                  - label: Ingress
                    name: ingress
              
...

**Comment by fimion** [comment]
(oversettelse fra google)

Hvilken versjon av staticcms bruker du?

**Comment by KaneFreeman** [comment]
Its possible this was fixed in [v1.2.13](https://github.com/StaticJsCMS/static-cms/releases/tag/v1.2.13) due to an update to the markdown editor. I am going to close this issue for now. Please reopen it if you continue to experience this issue.
